modules/sublimeBidAdapter.js

```diff
@@ -53,6 +53,26 @@ export function setState(value) {
   log('State has been updated :', state);
 }
 
+/**
+ * Get a notifyId from bid params or from sublime global
+ * @param {Object} params - The bid params
+ * @return {string}
+ */
+function getNotifyId(params) {
+  const sublime = window.sublime = window.sublime || {};
```

Comment:
Est-ce qu'on à forcement besoin de l'assignation sur window.sublime du coup ?
Cette formulation pourrait-elle suffire ?
```suggestion
  const sublime = window.sublime || {};
```